Source/Widgets/CesiumInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -159,6 +159,8 @@ define([
                                 if (picked instanceof Cesium3DTileFeature) {
                                     that._feature = picked;
                                     that._updateStats(true);
+                                } else {
+                                    that._feature = undefined;
```

Comment:
I noticed that features weren't being unselected due to changes in https://github.com/AnalyticalGraphicsInc/cesium/pull/5081, so this is a fix.